core/chains/evm/config/config.go

```diff
@@ -82,9 +75,29 @@ type ChainScopedOnlyConfig interface {
 	OCR2AutomationGasLimit() uint32
 }
 
+type EVM interface {
+	BalanceMonitor() BalanceMonitor
+	Transactions() Transactions
+}
+
+type BalanceMonitor interface {
+	Enabled() bool
+}
+
+type Transactions interface {
+	ForwardersEnabled() bool
+	ReaperInterval() time.Duration
+	ResendAfterThreshold() time.Duration
+	ReaperThreshold() time.Duration
+	MaxInFlight() uint32
+	MaxQueued() uint64
+}
+
 //go:generate mockery --quiet --name ChainScopedConfig --output ./mocks/ --case=underscore
 type ChainScopedConfig interface {
 	config.AppConfig
 	ChainScopedOnlyConfig
```

Comment:
So then the goal would be to dop this whole thing, right? Maybe leave a note, and/or deprecate?
```suggestion
	ChainScopedOnlyConfig // Deprecated
```
IIRC the linter will even object to using deprecated things.